packages/common/src/styles/RTLProvider.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { FC } from 'react';
+import { useRtl } from '../intl/intlHelpers';
+
+export const RTLProvider: FC = props => {
+  const isRtl = useRtl();
+
+  return <div style={{ direction: isRtl ? 'rtl' : 'ltr' }} {...props} />;
+};
```

Comment:
Maybe should be using a prop rather than the hook, as it couples the rtl implementation to how we're handling locales in this app pretty tightly

Reply:
yeah, just wasn't sure how a prop would help. still tightly bound